### DISCUSSION TITLE
Incoming emails are bouncing

### DISCUSSION THREAD

**Discussion by joshuacadman01** [postalserver/postal#2599] [Help with using Postal] [answered]
I have installed postal server quite a few months ago now, and now looking to use this for incoming emails. I have set the domain DNS details on the mail server, and followed the documentation online to setup the DNS records of the postal server. If i send an email to the configured domain, it bounces back with 'Remote server returned an error -> 530 Authentication required'. If i type 'post.domain.com' it loads the UI, however if I type the IP address into my browser, I do get the error 'You don't have authorisation to view this page. HTTP ERROR 403'

**Comment by sinnbeck** [comment, marked as answer]
What kind of routing did you set up?

**Reply by joshuacadman01** [reply]
That is where I was going wrong, I did not even see that tab. I added the HTTP endpoint and that was it, now there is a route, emails are coming through.
